db/migrate/20220125100834_add_external_id_to_submissions.rb

```diff
@@ -0,0 +1,12 @@
+class AddExternalIdToSubmissions < ActiveRecord::Migration[6.1]
+  def change
+    enable_extension 'pgcrypto' unless extension_enabled?('pgcrypto') # need to enable pgcrypto to use `gen_random_uuid()` function
+
+    add_column :submissions,
+               :external_id,
```

Comment:
**question:** What will `external_id` represent? My understanding is that:

- `id` will continue being the sequential, numerical ID
- `uuid` will be the non-sequential, generated ID

I assumed that we would expose the `uuid` field to clients as an "external" ID so I am wondering why we have a separate `external_id` column.

Reply:
@iskounen `external_id` is basically a `uuid` column you are talking about. Clients will be working with submissions using this column. Do you propose to rename `external_id` to `uuid`? I actually like `uuid` more

Reply:
Thanks for the explanation. I was reading the migration file incorrectly... 😅 

Regarding the name, my preference is to use `uuid` on the data model, and decide how we want to expose that to clients at the GraphQL layer.

Reply:
I will move this PR to WIP and rename db column. @kiryl-zubarau @iskounen what is your opinion on how to name this field in API?

Reply:
I would prefer `external id`